src/utils/parse-html.ts

```diff
@@ -4,22 +4,37 @@
 
 // polyfill and type definitions for String.prototype.replaceAll()
 import 'ts-replace-all';
-import ReactHtmlParser, { Options } from 'react-html-parser';
+
+// other libraries which have been tried and found wanting
+
+// import ReactHtmlParser, { Options } from 'react-html-parser';
+// import parser from 'html-react-parser';
+// import sanitizeHtml from 'sanitize-html';
+// import htmlparser2 from 'htmlparser2';
+// import parser from 'parse-entities';
+
+// no server-side, otherwise really good
```

Comment:
I should have said "no server-side without added hackery"

https://stackoverflow.com/questions/65646007/next-js-dompurify-sanitize-shows-typeerror-dompurify-webpack-imported-module